### ISSUE TITLE
implicit feedback

### ISSUE THREAD

**Issue by hangelwen** [mbhynes/als-ncg#1]
does this support implicit feedback?

From the command line options seems there is no option for `implicitPrefs`.

Thanks

**Comment by mbhynes** [comment]
No, it does not at current. The challenge is computing the outer product Y'*Y terms in the NCG component's line search for the implicit model. I would have to look at the algebra again, but I recall that it was a bit messy at the time.
